dask/array/reductions.py

```diff
@@ -333,8 +334,33 @@ def moment_agg(data, order=2, ddof=0, dtype='f8', sum=np.sum, **kwargs):
 
 def moment(a, order, axis=None, dtype=None, keepdims=False, ddof=0,
            split_every=None):
-    if not isinstance(order, int) or order < 2:
-        raise ValueError("Order must be an integer >= 2")
+
+    if not isinstance(order, int) or order < 0:
+        raise ValueError("Order must be an integer >= 0")
+
+    if order == 0:
+        # When order equals 0, the result is 1, by definition.
+        shape = list(a.shape)
+        del shape[axis]
```

Comment:
I think besides my comment here and below (and fixing the merge conflicts) this might be ready to go. What's the status of this @TomAugspurger?